app/views/aircrafts/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="container mt-3">
+  <div class="row">
+    <div class="col-12">
+      <h1>All our Aircrafts</h1>
+    </div>
+  </div>
+
+  <div class="row">
+    <% @aircrafts.each do |aircraft| %>
+      <div class="col-md-4">
+        <div class="aircraft-card">
+          <div class="aircraft-card-content">
+            <div class="aircraft-card-title">
+              <h4><%= aircraft.description %></h4>
+            </div>
+            <p><%= aircraft.aircraft_type %></p>
+            <div class="flex">
+              <div class="flex align-items-center">
+                <i class="fa-solid fa-people-line"></i>
+                <span><%= aircraft.departure_address %></span>
+              </div>
+              <div class="flex align-items-center ms-3">
+                <i class="fa-solid fa-euro-sign"></i>
+                <span><%= aircraft.arrival_address %></span>
+              </div>
+            </div>
+          </div>
+          <div class="flat-card-actions">
+            <%= link_to "More info", aircraft_path(aircraft) %>
+            <%= link_to "Edit", edit_aircraft_path(aircraft), class: "ms-3 text-warning" %>
+            <%= link_to "Remove", aircraft_path(aircraft),
```

Comment:
User should not be able to edit or remove any aircraft from hompage, please remove unecessary links to actions. Also remove the data actions.